### PR TITLE
[FIX] Use residual instead of debit, useful for partial payment

### PR DESCRIPTION
**Pull request by vrenaville**
* Purpose
We want to be able to resend a bvr with new amount if it's partially paid
* dev
replace the field debit by the amount_residual

### CODE REVIEW

File: l10n_ch_payment_slip/models/payment_slip.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def _compute_amount_hook(self):
         :return: total amount of payment slip
         :rtype: float
         """
-        return self.move_line_id.debit
+        return self.move_line_id.amount_residual
```
Comment (grindtildeath) [inline l10n_ch_payment_slip/models/payment_slip.py:135]: I'm not really sure if this is ok or not...  :thinking: 

On one hand, it makes sense the company wants to get paid only what's owed, but on the other hand there could be use cases where the company wants their invoices to be fully paid and manage overpayments with credit notes. 

Anyway if the company works with the latter case, IMO there shouldn't have been any reconciliations, so I guess it's ok to use `amount_residual` all the time...

**Review by grindtildeath — COMMENTED** [review]
Please increase module version number :)

We probably also want to forward port this to later versions.

**Comment by yvaucher** [comment]
@grindtildeath no need for module increase version, it can be done on merge by @OCA-git-bot

**Review by yvaucher — APPROVED** [review]
LGTM

About @grindtildeath concern, I think that amount_residual cover both cases.

**Review by grindtildeath — APPROVED** [review]

**Pull request merged**